test/integration/sftp/inspec.yml

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+name: sftp
+title: sftp proftpd formula
+maintainer: SaltStack Formulas
+license: Apache-2.0
+summary: Verify that the sftp configured correctly
```

Comment:
can you please correct this sentence with something like 'Verify that the proftpd sftp is setup and configured correctly'

Reply:
Ok I change the sentence